bitnami/jaeger/Chart.yaml

```diff
@@ -0,0 +1,30 @@
+annotations:
+  category: Infrastructure
+apiVersion: v2
+appVersion: 1.39.0
+dependencies:
+  - condition: cassandra.enabled
+    name: cassandra
+    repository: https://charts.bitnami.com/bitnami
+    version: 9.x.x
+  - name: common
+    repository: https://charts.bitnami.com/bitnami
+    tags:
+      - bitnami-common
+    version: 2.x.x
+description: Distributed tracing platform for monitoring microservices-based distributed systems.
+engine: gotpl
+home: https://github.com/bitnami/charts/tree/main/bitnami/jaeger
+icon: https://bitnami.com/assets/stacks/jaeger/img/jaeger-stack-220x234.png
+keywords:
+  - jaeger
+  - tracing
+  - monitoring
+maintainers:
+  - name: Bitnami
+    url: https://github.com/bitnami/charts
+name: jaeger
+sources:
+  - https://github.com/jaegertracing/jaeger
+  - https://www.jaegertracing.io/
+version: 1.0.0
```

Comment:
According to the template, we're normally starting in version 0.1.0 if I'm not mistaken

Reply:
[60a3da6](https://github.com/bitnami/charts/pull/13480/commits/60a3da67aa0fdfb7b11ed1e170effc28c585f97b)